java/flight/flight-sql/src/test/java/org/apache/arrow/flight/sql/example/FlightSqlExample.java

```diff
@@ -927,7 +954,14 @@ public Runnable acceptPutPreparedStatementQuery(CommandPreparedStatementQuery co
             .withCause(e)
             .toRuntimeException());
         return;
+      } catch (IOException e) {
```

Comment:
This can just be the same clause (eg catch (SQLException | IOException e) {})

Reply:
Yes, corrected.